flow/scripts/variables.yaml

```
@@ -607,7 +611,8 @@ FASTROUTE_TCL:
     Specifies a Tcl script with commands to run before FastRoute.
 USE_FILL:
   description: >
-    Whether to perform metal density filling. Default 0 (=off).
+    Whether to perform metal density filling.
+  default: 0 (=off)
```

```suggestion
    Whether to perform metal density filling. (0 means off)
  default: 0
```

this can keep defaults easily convertable to correct types